src/main/kotlin/FinalProject/Car.kt

```diff
@@ -0,0 +1,29 @@
+package FinalProject
+
+/* class for a single car in the simulation
+*/
+class Car(d: Boolean) {
+	/* variables
+	 */
+	private var drive = d
+	private var jammed: Boolean = false
+	
+	
+	/* getters / setters
+	 */
+	public fun getDrive(): Boolean {
+		return drive
+	}
+	public fun setDrive(): Boolean {
+		drive = d
```

Comment:
This doesn't compile. But the whole function seems to be unneeded, so it can be removed.